docs/UserGuide.md

```diff
@@ -19,8 +19,9 @@ ContactMate is a **desktop app for managing clients at Active Ageing Centres (AA
 1. Copy the file to the folder you want to use as the _home folder_ for ContactMate.
 
 1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar contactmate.jar` command to run the application.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+   1. When you first open the application, the list will be empty. 
+   1. After you add some sample data, the GUI will resemble the picture below. The next time you open ContactMate, the data will still be there for you to see.<br>
```

Comment:
Nice update to the quick start but also maybe could also edit the add command on line 30 to include the relevant fields?

Reply:
sure, i just added it